packages/grid/_modules_/grid/hooks/virtualization/useVirtualRows.ts

```diff
@@ -206,6 +206,8 @@ export const useVirtualRows = (
   const scrollingTimeout = React.useRef<any>(0);
```

Comment:
```suggestion
  const scrollingTimeout = React.useRef<number>(0);
```

Reply:
Ok, so I had the same issue with the `useColumnsReorder` hook. I guess because this is not inside a `.jsx` file the typings get confused and it says that this is not a number, the solution to this is to predefine it with `widow.`, then it works like this. And of course, the other solution is to type it as `NodeJS.SetTimeout` but that also looks a bit weird.

PS: I managed to fix it in the `useColumnsReorder` hook by removing the second param of the `setTimeout` but I was able to do it because for me it was `0`, here it has a value of `300` so we cant remove it. I suggest to leave it like this.

Reply:
> `.jsx`

We have forbidden this extension. The allowed ones are `.js`/`.ts`/`.tsx`.

Reply:
> And of course, the other solution is to type it as NodeJS.SetTimeout but that also looks a bit weird.

I don't think that we should ever use this approach. It's false. The type is always a number.